numpy/distutils/fcompiler/ibm.py

```diff
@@ -3,9 +3,12 @@
 import os
 import re
 import sys
+import subprocess
 
 from numpy.distutils.fcompiler import FCompiler
-from numpy.distutils.exec_command import exec_command, find_executable
+from numpy.distutils.exec_command import (
+    filepath_from_subprocess_output, find_executable
```

Comment:
`filepath_from_subprocess_output` is an extraneous import now -- I need to remove it as LGTM analysis indicates